code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -130,3 +130,8 @@
 		for(var/obj/item/weapon/bomberman/dispenser in src)
 			if(dispenser.spam_bomb)
 				dispenser.attack_self(src)
+
+		if(get_active_hand() && istype(get_active_hand(),/obj/item/weapon/rcl))

```

Comment:
People want it to place when you step. If you have an alternate suggestion, I'll entertain it, but this seems like the best implementation. If we de-snowflake by making an OnStep proc, that adds a bunch of unnecessary proc calls when only very few items (shoes, bomb dispenser, RCL) need it.